tests/logging_tests/tests.py

```diff
@@ -105,6 +106,47 @@ def test_django_logger_debug(self):
             self.logger.debug('debug')
             self.assertEqual(self.logger_output.getvalue(), '')
 
+    def test_page_found_no_warning(self):
+        with self.settings(DEBUG=True):
+            self.client.get('/innocent/')
+            self.assertEqual(self.logger_output.getvalue(), '')
+
+    def test_page_not_found_warning(self):
+        with self.settings(DEBUG=True):
+            self.client.get('/does_not_exist/')
+            self.assertEqual(self.logger_output.getvalue(), 'Not Found: /does_not_exist/\n')
+
+
+@override_settings(
+    USE_I18N=True,
+    LANGUAGES=[
+        ('en', 'English'),
+    ],
+    MIDDLEWARE_CLASSES=[
+        'django.middleware.locale.LocaleMiddleware',
+        'django.middleware.common.CommonMiddleware',
+    ],
+    ROOT_URLCONF='logging_tests.urls_i18n'
+)
+class I18NLoggingTest(LoggingCaptureMixin, SimpleTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(I18NLoggingTest, cls).setUpClass()
+        cls._logging = settings.LOGGING
+        logging.config.dictConfig(DEFAULT_LOGGING)
```

Comment:
No, I haven't tries that. I was actually just imitating the code from the DefaultLoggingTest earlier in the file.

It's not actually obvious to me how to use @override_settings to achieve what's needed here, but if you can give me some guidance, I'll be more than happy to try.

Otherwise the current PR should reflect all of your other recommended changes.